cadc-access-control/src/main/java/ca/nrc/cadc/ac/client/UserClient.java

```diff
@@ -149,7 +147,7 @@ public void augmentSubject(Subject subject) throws MalformedURLException
 
 	        // augment subject calls are always https with client certs
             URL usersURL = getRegistryClient()
-                .getServiceURL(this.serviceID, Standards.UMS_USERS_01, AuthMethod.CERT);
+                .getServiceURL(this.serviceID, Standards.UMS_USERS_01, getAuthMethod());
```

Comment:
How do you get a cookie for servops?  Only the servops cert is available on the web server hosts.

I suspect you are using the cookie of the user to get more information about the user, which is a getUser() call made by the user, not an augmentSubject call made from the privileged account.  If this is the case, you could be using the getUser() method instead.